debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/RetriableConnection.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.storage.jdbc;
+
+import java.io.UncheckedIOException;
```

Comment:
Looking at the code now is it even necessary to use the different exception?

Reply:
you are right